src/main/osd/osd_elements.c

```diff
@@ -684,6 +684,27 @@ static void osdElementPidProfileName(osdElementParms_t *element)
 }
 #endif
 
+#ifdef USE_OSD_PROFILES
+static void osdElementOsdProfileName(osdElementParms_t *element)
+{
+    uint8_t profileIndex = getCurrentOsdProfileIndex();
+
+    if (strlen(osdConfig()->profile[profileIndex-1]) == 0) {
+        tfp_sprintf(element->buff, "OSD_%u", profileIndex);
+    } else {
+        unsigned i;
+        for (i = 0; i < OSD_PROFILE_NAME_LENGTH; i++) {
+            if (osdConfig()->profile[profileIndex-1][i]) {
+                element->buff[i] = toupper((unsigned char)osdConfig()->profile[profileIndex-1][i]);
```

Comment:
add spaces around operator (this line and the one above).

Reply:
Ok.

Reply:
That went the wrong way, sorry. Please have a look at https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#spaces for the coding style rules pertaining to spaces.

Reply:
Ok, think I got it now... :o)